modules/dxp/apps/saml/saml-impl/src/main/java/com/liferay/saml/runtime/internal/SSOImpl.java

```diff
@@ -0,0 +1,57 @@
+/**
```

Comment:
When introducing a new class that implements an interface
        take into account:

* The class name should end with the interface name
* The package of the class matches the package of the interface



Refer to documentation: